timeseries/setup.py

```diff
@@ -29,6 +29,8 @@
     "pandas",
     "psutil>=5.7.3,<5.9",
     "gluonts~=0.11.0",
+    "torch>=1.9,<1.13",
```

Comment:
Are we sure we don't want to set this higher? Or is this inherited from other AG modules?

Reply:
The version cap is across all modules, 1.9 is from multimodal.